lib/const.rb

```diff
@@ -1,16 +1,19 @@
 # Defines common constants used in different parts of crew
 
-CREW_VERSION = '1.8.9'
+CREW_VERSION = '1.8.10'
 
-ARCH_ACTUAL = `uname -m`.strip
+ARCH_ACTUAL = `uname -m`.chomp
 # This helps with virtualized builds on aarch64 machines
 # which report armv8l when linux32 is run.
-ARCH = if ARCH_ACTUAL == 'armv8l' then 'armv7l' else ARCH_ACTUAL end
+ARCH = ENV['CREW_ARCH'] || if ARCH_ACTUAL == 'armv8l' then 'armv7l' else ARCH_ACTUAL end
+
+CREW_LIB_SUFFIX = if ARCH == 'x86_64' then '64' else '' end
 
 ARCH_LIB = if ARCH == 'x86_64' then 'lib64' else 'lib' end
 LIBC_VERSION = if File.exist? "/#{ARCH_LIB}/libc-2.27.so" then '2.27' else '2.23' end
```

Comment:
Is there a way to read what the libc version is and use that as the variable?

Reply:
There is no way to do it in ruby, but you can use a command to get it:
```shell
$ /lib64/libc.so.6 --version
GNU C Library (Gentoo 2.27-r20 p3) stable release version 2.27.
Copyright (C) 2018 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.
There is NO warranty; not even for MERCHANTABILITY or FITNESS FOR A
PARTICULAR PURPOSE.
Compiled by GNU CC version 4.9.x 20150123 (prerelease).
libc ABIs: UNIQUE IFUNC
For bug reporting instructions, please see:
<http://crbug.com/new>.
```

Reply:
We could probably use sed and tr to get it

Or we could just tr the string of the name of the library.

Reply:
Change made